src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java

```diff
@@ -532,6 +534,14 @@ public void siControllaLaTabellaDelegatiDallImpresa() {
         delegatiImpresaSection.checkTabellaDelegheDellImpresa();
     }
 
+    @And("Non si inserisce il codice OTP e l invito della delega non è più presente")
+    public void nonSiInserisceIlCodiceOTPELInvitoDellaDelegaNonèPiùPresente(){
+        logger.info("Si attendono 15 minuti e 15 secondi per la decadenza dell accettazione della delega");
```

Comment:
There is already a log that prints the time it'll takes:
https://github.com/pagopa/pn-frontend-e2e/blob/77af69f1e7506355cfe7ad84532f78ac16b3d03b/src/test/java/it/pn/frontend/e2e/utility/DataPopulation.java#L79-L85